x-pack/solutions/security/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_trends.ts

```diff
@@ -45,11 +43,31 @@ export type Trends = Array<{
   clusters: Record<string, Stats>;
   benchmarks: Record<string, Stats>;
 }>;
-
-export const getTrendsQuery = (policyTemplate: PosturePolicyTemplate) => ({
+export interface ScoreTrendAggregateResponse {
+  by_namespace: {
+    buckets: Array<{
+      key: string; // namespace name, e.g., "default"
+      doc_count: number;
+      all_scores: {
+        hits: {
+          hits: Array<{
+            _source: {
+              '@timestamp': string;
+              total_findings: number;
+              passed_findings: number;
+              failed_findings: number;
+              score_by_cluster_id: ScoreByClusterId;
+              score_by_benchmark_id: ScoreByBenchmarkId;
+            };
+          }>;
+        };
+      };
+    }>;
+  };
+}
+export const getTrendsQuery = (policyTemplate: PosturePolicyTemplate): SearchRequest => ({
   index: BENCHMARK_SCORE_INDEX_DEFAULT_NS,
-  // Amount of samples of the last 24 hours (accounting that we take a sample every 5 minutes)
-  size: (24 * 60) / CSPM_FINDINGS_STATS_INTERVAL,
+  size: 0,
```

Comment:
we don't need the documents from the query but the documents from the aggregation.